docs/getting-started-guides/kubeadm.md

```diff
@@ -165,7 +167,8 @@ also support [Network Policy](/docs/user-guide/networkpolicies/). See the [add-o
 
 You can install a pod network add-on with the following command: 
 
-    # kubectl apply -f <add-on.yaml>
```

Comment:
We really need to come to some agreement on this section of the doc - it can't just keep going back and forth as people add new PRs.

@thockin @chrismarino